content/anchor-frontend.md

```diff
@@ -0,0 +1,619 @@
+- `instructionDataInputs` - include any instruction data required by the instruction within the parentheses after the instruction name
+- `accounts`- list of accounts required by the instruction being invoked
+- `signers` - any additional signers required by the instruction
+- `rpc` -  creates and sends a signed transaction with the specified instruction and returns a `TransactionSignature`. When using `.rpc`, the `Wallet` from the `Provider` is automatically included as a signer and does not have to be listed explicitly stated.
+
+Note that if no additional signers are required by the instruction other than the `Wallet` specified with the `Provider`, the `.signer([])` line can be excluded.
+
+You can also build the transaction directly by changing `.rpc()` to `.transaction()`.  This builds a `Transaction` object using the instruction specified.
+
+```tsx
+// creates transaction
+const transaction = await program.methods
+  .instructionName(instructionDataInputs)
+  .accounts({})
+  .transaction()
+
+await sendTransaction(transaction, connection)
+```
+
+Similarly, you can use the same format to build an instruction using `.instruction` and then manually add the instructions to a new transaction. This builds a `TransactionInstruction` object using the instruction specified.
+
+```tsx
+// creates first instruction
+const instructionOne = await program.methods
+  .instructionOneName(instructionOneDataInputs)
+  .accounts({})
+  .instruction()
+
+// creates second instruction
+const instructionTwo = await program.methods
+  .instructionTwoName(instructionTwoDataInputs)
+  .accounts({})
+  .instruction()
+
+// add both instruction to one transaction
+const transaction = new Transaction().add(instructionOne, instructionTwo)
+
+// send transaction
+await sendTransaction(transaction, connection)
+```
+
+In summary, the Anchor `MethodsBuilder` provides a simplified and more flexible way to interact with on-chain programs. You can build an instruction, a transaction, or build and send a transaction using basically the same format without having to manually serialize or deserialize the accounts or instruction data.
+
+## Send Transactions
+
+As a refresher, we use the `sendTransaction` method from the `useWallet()` hook provided by `@solana/wallet-adapter-react` to send transactions with a wallet adapter.
+
+The `sendTransaction` method prompts a user to approve and sign a transaction before sending.
+
+```tsx
+import { useWallet } from "@solana/wallet-adapter-react"
+
+const { sendTransaction } = useWallet()
+```
+
+If the transaction was build using either `.instruction` or `.transaction` from the Anchor `MethodsBuilder`, you’ll need to send the transaction with `sendTransaction` to prompt a connected wallet for approval.
+
+```tsx
+sendTransaction(transaction, connection)
+```
+
+If the instruction requires additionally signatures, you need add the additionally signatures to `sendTransaction` using the following format:
```

Comment:
If the instruction requires additional signatures, you'll need to add them  to `sendTransaction` using the following format: